picasso/src/main/java/com/squareup/picasso/RequestCreator.java

```diff
@@ -354,12 +355,16 @@ public Bitmap get() throws IOException {
   }
 
   /**
-   * Asynchronously fulfills the request without a {@link ImageView} or {@link Target}. This is
-   * useful when you want to warm up the cache with an image.
+   * Asynchronously fulfills the request without a {@link ImageView} or {@link Target},
+   * and invokes the target {@link Callback} if it's not {@code null}. This is
+   * useful when you want to warm up the cache with an image,
+   * Use this when you want a callback after the image has been processed.
    * <p>
-   * <em>Note:</em> It is safe to invoke this method from any thread.
+   *<em>Note:</em> The {@link Callback} param is a strong reference and will prevent your
+   * {@link android.app.Activity} or {@link android.app.Fragment} from being garbage collected. If
+   * you use this method, it is <b>strongly</b> recommended you invoke an adjacent
    */
-  public void fetch() {
+  public void fetch(Callback callback) {
```

Comment:
We still need a version of `fetch()` that takes no arguments. It can just call `fetch(null)`.